tests/cli/test_nb.py

```diff
@@ -82,6 +82,47 @@ def test_format_skips_non_notebooks(monkeypatch, backup_simple,
     cli.cmd_router()
 
 
+def test_format_adjusts_pipeline(monkeypatch, tmp_nbs):
+    monkeypatch.setattr(sys, 'argv', ['ploomber', 'nb', '--format', 'ipynb'])
+    assert Path('load.py').exists()
+    cli.cmd_router()
+
+    assert jupytext.read('load.ipynb')
+    assert '.py' not in Path('pipeline.yaml').read_text()
+
+
+def test_format_same_pipeline(monkeypatch, tmp_nbs):
+    monkeypatch.setattr(sys, 'argv', ['ploomber', 'nb', '--format', 'py'])
+    pipeline = Path('pipeline.yaml').read_text()
+    cli.cmd_router()
+
+    assert pipeline == Path('pipeline.yaml').read_text()
+
+
+def test_format_no_entry_point(monkeypatch, tmp_nbs_factory, capsys):
+    monkeypatch.setattr(sys, 'argv', [
+        'ploomber', 'nb', '--entry-point', 'factory.make', '--format', 'ipynb'
```

Comment:
I think `factory.make` isn't working because the module isn't in the path. so if you go with `test_pkg` you can delete the `nbs-factory` code and fixture.